docs/reference/setup/important-settings/path-settings.asciidoc

```diff
@@ -26,3 +26,112 @@ Supported `path.data` and `path.logs` values vary by platform:
 include::{es-repo-dir}/tab-widgets/code.asciidoc[]
 
 include::{es-repo-dir}/tab-widgets/customize-data-log-path-widget.asciidoc[]
+
+[discrete]
+==== Multiple data paths
+deprecated::[7.13.0]
+
+If needed, you can specify multiple paths in `path.data`. {es} stores the node's
+data across all provided paths but keeps each shard's data on the same path.
+
+{es} does not balance shards across a node's data paths. High disk
+usage in a single path can trigger a <<disk-based-shard-allocation,high disk
+usage watermark>> for the entire node. If triggered, {es} will not add shards to
+the node, even if the node’s other paths have available disk space. If you need
+additional disk space, we recommend you add a new node rather than additional
+data paths.
+
+include::{es-repo-dir}/tab-widgets/multi-data-path-widget.asciidoc[]
+
+[discrete]
+[[mdp-migrate]]
+==== Migrate from multiple data paths
+
+Support for multiple data paths was deprecated in 7.13 and will be removed
+in a future release.
+
+As an alternative to multiple data paths, you can create a filesystem which
```

Comment:
Everything below this line was copied from the migration guide. The copied text is unchanged.